src/ralph/accessories/models.py

```diff
@@ -118,10 +112,16 @@ def restock(cls, instances, **kwargs):
     )
     def release_accessories(cls, instances, **kwargs):
         user = get_user_model().objects.get(pk=int(kwargs['user']))
-        AccessoryUser.objects.create(
-            user=user, quantity=kwargs['quantity'],
-            accessory_id=instances[0].id
-        )
+        accessory_user = AccessoryUser.objects.filter(user=user, accessory=instances[0]) # noqa
```

Comment:
This could be simplified: 
```suggestion
        accessory_user, created = AccessoryUser.objects.get_or_create(
            user=user, 
            accessory=instances[0], 
            defaults={'accessory_id': instances[0].id}
        )
        if created:
            accessory_user.quantity += kwargs['quantity']
            accessory_user.save()
```